src/app/money/import/mappings.ts

```diff
@@ -291,3 +296,9 @@ interface N26Row {
   "Fremdwährung": string;
   "Wechselkurs": string;
 }
+
+interface Generic_Eng {
```

Comment:
Please follow naming conventions of the other interfaces. I suggest `GenericEngRow`.